Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:13.8.0 as react-build
+WORKDIR /app
+COPY . .
+RUN npm install && npm run build
+
+FROM node:13.8.0
+RUN yarn global add serve
+WORKDIR /app
+COPY --from=react-build /app/build ./build
+CMD ["serve", "-s", "build", "-p", "3000"]
```

Comment:
Remember that for a "finished" image that's going to be run it's more common to use `ENTRYPOINT` (e.g. see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#entrypoint). So here I'd do:

```
CMD [ "-p", "3000" ]
ENTRYPOINT ["serve", "-s", "build"]
```

that way it's easy to start the server but also very easy to specify a different port.

Reply:
Neat--I never knew you could use both CMD and ENTRYPOINT to run a single command with multiple flags. Very cool!